runtime/compiler/control/J9Options.cpp

```diff
@@ -1448,21 +1448,22 @@ J9::Options::fePreProcess(void * base)
    // fePreProcess is called twice - for AOT and JIT options parsing, which is redundant in terms of
    // processing the -Xlp:codecache options.
    // We should parse the -Xlp:codecache options only once though to avoid duplicate NLS messages.
-   static bool parsedXlpCodeCacheOptions = false;
+   static bool parsedCodeCacheOptions = false;
 
-   if (!parsedXlpCodeCacheOptions)
+   if (!parsedCodeCacheOptions)
       {
-      parsedXlpCodeCacheOptions = true;
+      parsedCodeCacheOptions = true;
 
       UDATA requestedLargeCodePageSize = 0;
       UDATA requestedLargeCodePageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
       UDATA largePageSize = 0;
       UDATA largePageFlags = 0;
       int32_t xlpCodeCacheIndex = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, "-Xlp:codecache:", NULL);
       int32_t xlpIndex = FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xlp", NULL);
+      int32_t largePageArgIndex = vm->largePageArgIndex;
 
       // Parse -Xlp:codecache:pagesize=<size> as the right most option
-      if (xlpCodeCacheIndex > xlpIndex)
+      if (xlpCodeCacheIndex > xlpIndex && xlpCodeCacheIndex > largePageArgIndex)
```

Comment:
please do not rely on order of operations, add extra pairs of brackets

Reply:
Thanks. See https://github.com/eclipse/openj9/pull/7476/commits/cad6beaec1cc182b1a5325f005ef62dba56d2810

Reply:
Thanks. See https://github.com/eclipse/openj9/pull/7476/commits/cad6beaec1cc182b1a5325f005ef62dba56d2810